services/fdb/fdb.proto

```diff
@@ -276,6 +277,9 @@ message FDBCLIKill {
     FDBCLIKillAll all = 3;
     FDBCLIKillTargets targets = 4;
   }
+  // fdbcli may return before message is actually sent.
```

Comment:
Could you add a comment describe what this does? Something like "Duration to sleep after sending the kill command"